hackbg/chainlink-keepers-templates
# Batch NFT reveal contract

How to manually test:
1. Deploy via hardhat console
2. Verify contract (needed to register upkeep from app)
3. Fund contract with LINK (needed to pay for VRF calls)
4. Register upkeep (https://keepers.chain.link)
5. Mint some tokens
6. Check if revealed when criteria is met
7. Check collection on opensea (works only on Rinkeby)
